### PR TITLE
paragraphs in messages

### LINKED ISSUE #1334: Paragraphs in messages
Subject: Paragraphs in messages
Description:
Sent a message with paragraphs formatting but when the message sent the formatting was not saved.

Results:

Backend version: develop-9b18c584
Frontend version: develop-9b18c584
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36
Page: https://app.couchers.org/messages/request/175
User (spoofable): tsab (467)

### PR DESCRIPTION
**Pull request by fmanuelos**
closes #1334

the components which display messages was updated with the right white-space css set

after:
Screen Shot 2021-06-24 at 2 24 00 PM

before:
Screen Shot 2021-06-24 at 2 24 32 PM
 

**Frontend checklist**
- [x] Formatted my code with `yarn format && yarn lint --fix`
- [x] There are no warnings from `yarn lint`
- [x] There are no console warnings when running the app
- [ ] Added any new components to storybook
- [ ] Added tests where relevant
- [ ] All tests pass
- [x] Clicked around my changes running locally and it works
- [x] Checked Desktop, Mobile and Tablet screen sizes

### CODE REVIEW

**Review by lucaslcode — APPROVED** [review]
CI failure is unrelated

**Comment by aapeliv** [comment]
@fmanuelos: once a PR is approved, please merge it yourself when you get a chance!

**Pull request merged**
